examples/gnrc_networking/Makefile

```diff
@@ -20,8 +20,8 @@ USEMODULE += auto_init_gnrc_netif
 USEMODULE += gnrc_ipv6_router_default
 USEMODULE += gnrc_udp
 # Add a routing protocol
-USEMODULE += gnrc_rpl
-USEMODULE += auto_init_gnrc_rpl
+USEMODULE += aodvv2
+#USEMODULE += auto_init_gnrc_rpl
```

Comment:
@Lotterleben please revert this change, it was (wrongly) introduced by my commit. I did for testing, before making a separate copy into `example/gnrc_aodvv2`.